src/test/java/awesomesyd/fishingexample/BarbedEnchantment.java

```diff
@@ -0,0 +1,31 @@
+package awesomesyd.fishingexample;
```

Comment:
No, you cannot have your own name and package for the tests. And a single test mod should all be in one file. Also, you need to run `licenseFormat` so that the license is applied.

Reply:
ran gradlew licenseFormat, it also put the forge license on a few things that I never touched.